Improve types (#25)
pawcoding/astro-loader-pocketbase: README.md

It's recommended to use e.g. the title of the entry to be easily searchable and readable.
**Do not use e.g. rich text fields as ids.**

### Improved types

> By default PocketBase reports `number` and `boolean` fields as not required, even though the API will always return at least `0` and `false` respectively.

Reviewer: I think the line is a bit misleading, at first I thought `0` and `false` are the returned values for the requiredness.
I therefore suggest slightly improving the sentence:
```suggestion
By default PocketBase reports `number` and `boolean` fields as not required, even though the API will always return `0` and `false` respectively if the concrete fields are not set.
```
